### ISSUE TITLE
Kit:  Better Error report

### ISSUE THREAD

**Issue by zhihengGet** [sveltejs/kit#11169]
### Describe the bug

TLDR:  kit does not report bad imports in console.

hopefully, it is not something with my kit setup or machine

 sometime when i go to a page, it just returns an black page with 500, +error.svelte couldn't catch anything, no err in console nor chrome console....

The problem were with one of my .svelte file, when this .svelte imports a file that actually a type import {type} from "...", vite return error saying "type" does not exist, we need import type {type} from "..." but kit does not show this error !!  i had to bisect through my imports to find it...

plz fix , and i feel like there are other cases that can cause this to happen with regards to imports statemetns, no error, just 500 .. but couldn't think of it rn...

if go to my repro, stop app then restart it, it goto 500

### Reproduction

https://www.sveltelab.dev/?files=.%2Fsrc%2Flib%2Ftype.ts#code=N4IgdA5glgLlEDsD2AnApiAXKAZlANhtiAMZIIxoUDOWoA1lAgCZYjUBua%2Bl%2BAhgCMAtAFcoADj4gANCAEicONCiwBtACwA2aZvHTxAZmkBOAKz6jARkvbrl6dfUOADPcvOH7z2%2Bcf39sxdjbxc-SwB2Fz1-T3MvdUjjaIiXOOdo3xdpBOzbSzjLDJ9k21dstMj3AtLsyssAJhNE4PdEyucrMvjSty9rDNyXXrC9LWzGrwMjMZjrEcHZ%2Bo9jdpaGqKGs-tTPHsHjNeGXCec06dK88xmbHfcWrvzdpqDPRvUrk88M6s2xg5C5sdbh0FjdrFc8qU0vdej8Vi9rG8Pi4ALoAXwxsjAAAd0DA4MoUCQ6CA8IQSWQKFQYLRiIwWGxONxeIJRBIpLJ5IplGoGp0TNIDE5rHU0vZxE54UlYoKnKY9AYJmDLMKUjF1MLlrKZVVPH4Qe4ivpenVenshdJ5YKlXlVe17BqssELdZbc9dYbbn0QZpTQjnMF4RarYrtYC7HrNhbHV5naqlZqCmFtOJ2mlky4nMGFW9gupRqNNQKXetPSKXJ0kzLs9aTPYSwnNq6nh6Hk53q8NuX3JXIx3wV8m%2BkW2ULcZpkW452y2qDadI4ZNvDHCZ7k5dF7VeYa6HjPX29oxxOnWHrIPZirPInQk3HjvGnuHPUjKGXc4TmkjQP79r1G9LueXTDs2MTRlmjQlr0Npekc35yjmT4vhB8bupeA4Xlm8G1iWBRfsBYKjuBT7bhM9TmOOWxkaiGJorI2J8CQ9B8BAaBgAAVtQ5AkmSRCgJSlA0CS9KsJg7BcDw3CsmIkgyHIChKComCqHyJ4lssqxRlhr6Snoy6PLq-xgapqpfmh%2BnzhGgI-tGozaPmgzqFOMZag2V5erpdTmlp0Ert2ZTOcWW6rm5Hqlsq27eYhJ5Tu4%2Bq9AqkUlt8I71iccWDkZsaBSYyRpl2mTWSZXrmg%2BuXAhkHiZVqMVhZq8XAr0U4hshgE-KBSqNheyUXk5k7ZWWFl9DchXTmhDyQg1bmhh2-alvh7S9oCs1iqhkpruUN6hURYzTVc5HrTE-ytLWO3tn1MXvgCY2Bp52rNaenXjSOK0rrtG2Il2s6LfEBQrXpa2Zu9mRbSdh5Iu9lhrNeIGjv%2BYZag0Vx9W%2B6XbBuF4nLBwMPMs9wvXekUqVlU6aEF2yzPhezuABOTLpkek%2BW2G16bdEYjcYcoeGM9ncwl2TndqpM6oUQ4lEONPtHlzbs%2B2ebaKYXO2ZaLUBYGD0hU9MP45hloIWOnODKYdl2Xzqv9bVt4Ta2vTtgUJxlT2Xpc79gHFCld165KBvc3mSuOb%2BAs4a7n1ebr2Fe8rDlK6YL5ndFp7dQR%2B5h9pJhynDpgZ5VfWI-zxktvUUGaSnLV3ACwOzkaVX9djRcXvqqOxnjt4RSXUXE6euGi%2B7HZHc4dXBcdMuWg68sZzrWW530VHuOimLsESJLMFA6AkDAqAAJ4knw2LYmAAAWMAALb4NxBC8aQ5ACTSQlMCJYnMpJwjSRycncopqiaJVRgbuEwS%2BvoJw4hgjiA8JoZ0JxVSlHuHoTQnwv6AxhjA2sno9L6h-sneugtPj-31APOmYCcECgQWXC82ggwPnWv9HUT1NCYLVARcGRttTTUiHA%2BOJDgKDX1JEUMvkgKwMwfOI6Z4rJEXcKqAe7hGh0JYaKYOM4qbOCljcP48j0KazvJEHI0jULJREUca4Sp0EsLeGwohuD-RmmeHw2MGkYiyKSpNawawlQ2yeBaXchxIww1gjcRxqpm6xRHOuHwjd9EwghkEiupQdasJ0BYwU6iRYUzFhTFRxtEFlAIUkk8CCdH9yXO0QheTljJDCU%2BVKN49LkO0S7cmAj3bHVCnZO2Gs3YRhyf2emt14QBMQaIlJTMLTsI7pw-UXdbFtULvlew-TgmhTaagyoMz7oGJ8UnDZAERkQWSXhNJQEMn7HKZtGZBg2Fc2Ke5BJpTag5XLn4GZoywGXIGdAjYzy560RAPgKAAgl4rzQGvTeJIADCSARBUhQGAJkEkz7kmIPxaktIGB30ZOJFkz92SyS5ApNQJDHgiOhmFbQzyBStlcVdKBKDnweNIiRIEvlREBnDg5VmMyPqgRzlw6xXirp1AzKGRxbw9CmHuNhCCRCDDywQq2SV2Rcb8q2bWdhwrsi%2B2yFUom1V3Q8pCqo5Y5DvHdiGmDL0DLgloIrLeVUJoaHRDOZA4EnQsxGGMHZewYr47rJNYKmRVTTCRC9ZPGe%2BpyWXV9YPAiBQjh8MKVahZeqAEDX2i4bRXMtW0o7j6hhfrsgYJOsEYNCNQ06FeQOIlXpoJkq8Ag6UJqraDMqm4yMVo%2B4RLTToNwWb40Cs2km9oHbmlUTJf-QdOw%2B72hvHw2lOa%2B0gVLS0Wdxrc10q7eutV-shVIUFGsKEVjd21nugYPdwJInfiXWkMtdz60MMbUcT5NFZB3zQAAD3Yii0k58KRX2RbfBkolYWYrZDJTk8keRKRyDkWx9tnh9y1RGiyervykV6Ha2xN1lW2LyAh6Eo13HS0ZpIpVjsJEoIIgPXch5Sj6N0maum%2Bj5FxqrNhrJLs40toGo3A81F54wA3tiViN9iA8R-VSQSdI0WAYxU-EDr9cUQeUl0JY%2BH9VmoXXXWGa7%2BlbvUF8%2BeKAIWUE-cvVe68UBb2IAAan4BvIzMKZPwovkiiTqKAMPwkvwLFoG354qUgS1N-Cln%2BKIRS1TTKzUnrZdamcAQPIbD%2BGsdjRbxX5PkZW0K1bPi1rCJ%2BK6ZqciPvnlZ%2BiLEYXKC4NCjiTmxPX0-cJdFj8vNyZxeBj%2BiwwicbMvRlduW12xd1c29du51EcqbidDNKq5nyu1eSsa1iTWqiG7NtWgI6aROPf-WpNDU2hglEOSlyz3ZRZDeYYtxFLT3C%2BbIErzFWJAYvqJxFv7XMgAa9JprUlsVgffvivrGXBpZZPGFzlgzqVCr9JG3xU0TidWZRAw90r8rkVGGwyHq7ocrmuF3VJx2UtbARspzrVL%2BvIeGmlVC23ztpZlAD4LpLssvMBv7Ud0hwiY3yicmdnR1JPHhNcXnpDlPbn5DTsjLPGfXuTYM%2BdmzXpmJud6a9H1WfhD2NQqyJwV2UYfIkE5DbyNKMO8OGpLdaym6ynW3hTr%2BE%2BH9fDalIEwpndR3chZZ4REQmFvsppJu9cLFroOHRcuWWcr900gmhvVoxdocnQI9bw-hmUdglBFcAhJdBDjxpmPw-ShmIH7YwejfTkT3Lu8MjXl9PgeW9YRXvkQHwEgAQfB8BgBINQT9T2%2BIveE25%2B%2BD3msv1a79pSVomVof6z2gUM16nJ97vFydQNIoK2yKKuU4q8x7RoXU%2B5Up7g79MA6NIZNhz3XO5viGFRmYL-3xtf290yMaNGIrPO00FTaO2hq%2By5zNWXZPDP7fa-ODW-DsFfB-QpJ-BVVfV-EVXJR0aaL-d-X-c-O-WfHfaUPfTtUAz1WVCA8yZ-S0F-UwOOUMYgyOH-eAxoQNS0Q8bRT1DfVAwA%2BfYArAs7Kg3Ak-AglfeyMgygyOTOOAoiagw-OA%2Bg--LfDRdAm-Vgy0IwcAzgqA7gtfGAwg6AigoQk2UQv-LKAAyQiGKmFTZsGaItDg%2B1KAnmbaJAvg%2B-esOg7QrUXQ8yHfTHQw1RM7DweQsw9QF-awqwoQjw7cOwlAxwq-WaAwjMbAtuR-fAxQjw2Avg5hdQo9Kg3ZZAhgkImQmGKRUPfyK4Ew1PBQ7w9VWQlQ-2HmPwo9OQl8II9IiQpw-QrJHwSJUAjw0wyAoosA%2BIjQwQ5I8gmo8Qy-TIijA7X4M7HAgorwjwogt4T-aA7-D-I9OUWwtIgY5aTIiNJtJ4TbNnNYUyFCPSZLUo8iOoDqWZQYhoWEX3PMIiTQGjJ2TtcINoYKRcC44WYvcIMIPxOyIiKLF4oCKwZlKpcQcwcIF8a42scQFtW9BbM8Pgv%2BJ8R5HwToQxIiYE%2B5KHZUBYsZJ48MG4DXbPYcPgqLD4zcGxRoRcBNO8cE18f%2BZIW1YEIbCk2XbcakiCYIdnbuBpBbYRTDUMcIUyKdKLPgitZk84pmOVew6fOo0ItILrCIvDSyIacYmktnCZNyChNndHT4w9YU0PTYxUtpTAmIc7KeMNIWIXCpAya3KA0HbIjnJlLfH1fCdxNYvubIjbXAwXMaOw7nE8MfMyCfQjduHVDI46Zgo0ufPIqIvAvLIoiwk6CohAtQ-ojuUMyM3fZoGQsAtomIuMwsBMnopMxIlMkM6UmQjAzMsMsY6Mwonwywws3MPolY1MssqsjMwedMgQzw9ousgspI0gqYrQlAi-NYtsisjsg-OQnM2M3st-BsyOeY5s0ssU5wuXVw1pYo7s3M2c2A-sqg7RKwsQlslchojrJ4FomsyYuY%2Bsvcmg5MpcgUEctA08onc86srcmc68vshY0guUaoh8y9AnR8-XYGR1YMx8qNDpMHQYt0kY40z06PY6FA00i7JXaE05TNfkQMaQ5pA0AMzwedTIXuY1Z0tyB-EWahOM1s9M8cxfSI9QI86qEi-XS8ci%2BLLHZ2E8sIpsdcu-PMRi8NTDfhb6VogssYUwM7ZEl4ZCjYr4oGXXf0PLV4ti6PKiri2Uq6CIgSzDf4eDZVYi8uFiwmCiuodsTi0c9MrIuC3I4ohiyU00-TJ9dgGAPgOAYkYgUzIFczSzXAPgDgKASkHEBACAGrZ7cTXvN7KTDzYDIfH7PzVQZ8FoR8QoEBDwfktnNwfkeoGoDwXKhwIwUEtnGRPQSE6QPKrmbQcq38QUMq2qjwRKhwZK4BfQcwBqyGNwZqhoSIKqqYfQNKlMdcYIKqtqpq-%2BJweqk7UanQca6QEajqsa2quahwZqoWCapK%2BanQcwNaxqja1a2aya3ara-a9a5a-%2BI6palK6axag606q67OQBY6na26lIKqwZFMRodKxcba9q26i65qv6hanIAG7QkWYGsGj62ayIcIewKqylDkzqx4PKy8YGpGpwYG%2BwYG9QKwNG7quqx6n6y6wqjcTQcwfbBob6qasGm6y69GyITGh1fGqax8d1Rmjaqmk6y6imja9Ydmp6mm6m5qkQmZXmgmzqowEWqaywcWvQYGgQwq3q3yeoNki7CWtmgW4IYG%2BE1W36yUUiUocW9W2ajGw2kWVqw24GqwRoeoEqm2zwAEq2-q1m56yGJ2y6hoK2w2umk22lbWy69hWW3W66jm-6s7c6w28QOZLm3682mO4OvMaYOQr2uOmUJwDkmOV2-6425OsGlZKOt2s27O72tGk2y8LGjOjW2Ovmzq4u5O%2BEsugwQc%2BE0BBEqKR48u9u9G4cX2-6820seElICSyOaGhwWaiyAwLOquiu5O9CZGyu0W5uYG8BS0Auyexq4UNulcMGqWhwMu7updJE9dAcL1dG2lDsPe9uiMdGs8c%2B6UQe2aSIMmlTNE8%2Bj6YwD2wu6ellE%2Bie%2Be3-DcKiF%2B4Ww2sYc%2BlfKLQuSoH-c%2BxcTGzMqiJHDk4G-bAwF2Tu2XWOHQA25O3eueym3BtWuIfsYNa%2Bl2-IWBTi8%2BzQLB1eqqtKn%2Bdk2BGWw224kwbQAGxy2QB7NvcgPACAD9UK7vcK%2BrKKgfL7HzBTdrJDQ7aCoMnJBmCPcRDY%2BksuFU80-SIZKCpRCWQbVCRmF044gilsM8KLE015P8cYFVR3MbDnX0kHMKAjA1Q9NRrPTR8WFOyWYqHU0eHQLUxlJDS2hlcxgpSCjZH4PlRU-UD8a1IVTC4HZPFpKIqfNWKDa0uRk4hR5mOoVmYxuUrsMCme9J0HBx7ZJUGNA9ONPxROZR70c9U1FO8RcyHwBDR3ZUUpsOFJztBJ7HBRSmX4UocJN3NPKNK4HEnpUaKmUugxg3NmWHEnEbVPROH4fPCCjpvkzoGbMbO1eXW8eYEGCJg9XE9TQaC1JU-rdRwp%2BxtTIcZbaCmdQMlIKZS2EcPCG52NBSx5sPdPYEKJwye0oaKpzSg5up4U9RoJPE9aWxLZ8nEJKMBSy1C5iZnjVM7Ra0v4ppiYp3PINpq0Dp3CpZgCLkkOPp-teqApbUz3G9K5eRzHMyC1WSmHBEGRrTUKSJknMuC3QaN5uDFi26eZ%2BF8Zm1FOmoNWNJgVjCNuUV0HRF8wnwclUtXOUxni4nRU8HCnCmOFyuNdHNVl1sOQq7Jy-yygbhhAXh-hkTb9MKurf9fvGTQfb7XzRTOxrrVVqKD5jRol9JTxmdOluHJo7CBamen3HPbRvF88uoROc0fHb0BUy5iLGlHdRNO3e4hxXx3RXVmlH1-VP1lUhpNqUcKNsNM8opq5uNVGS6FmIl0tocKJpPOZVNxDM4G0pJkvB2OcKsI%2BhCZK9RoNkPI5OMgIfkXOOy4tfTIAA

### System Info

```Shell
windows
```

### Severity

serious, but I can work around it
